hnn_core/network_models.py

```diff
@@ -35,9 +35,10 @@ def jones_2009_model(params=None, add_drives_from_params=False):
     Network is composed of an all-to-all connectivity pattern between cells.
     """
     hnn_core_root = op.dirname(hnn_core.__file__)
-    params_fname = op.join(hnn_core_root, 'param', 'default.json')
     if params is None:
-        params = read_params(params_fname)
+        params = op.join(hnn_core_root, 'param', 'default.json')
+    if isinstance(params, str):
+        params = read_params(params)
 
     net = Network(params, add_drives_from_params=add_drives_from_params)
```

Comment:
Is there a test for what happens when something other than a `dict` is passed here?

Reply:
there wasn't. Added here: afab4aef29a2a0eb1d4ad54e122ae3407bc1f2cf